intra-factory_transport/requirements.txt

```diff
@@ -0,0 +1,20 @@
+# Core dependencies
+numpy>=1.20.0
+pandas>=1.3.0
+matplotlib>=3.4.0
+scipy>=1.7.0
+
+# Jupyter environment
+jupyter>=1.0.0
+notebook>=6.4.0
+ipykernel>=6.0.0
+
+# Visualization
+seaborn>=0.11.0
```

Comment:
Don't think we need seaborn and few other requirements, lets remove what not needed.